tests/ui_tests/test_admin_page.py

```diff
@@ -88,7 +88,7 @@ def test_settings_filter(self, settings_page: AdminSettingsPage):
             settings_page.config.clear_search()
         with allure.step(
             f'Click on {params["group"]} group and check {params["field_display_name"]} '
-            'is not presented after group roll up'
+            f'is not presented after group roll up'
```

Comment:
nothing to format here, typo, I guess

Reply:
it is two lines f-string

Reply:
but in the second there's nothing to format or am I missing something?

Reply:
removed